src/plugins/intel_cpu/src/mkldnn_plugin.cpp

```diff
@@ -176,6 +177,7 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
 
     static const auto precisions = get_convert_precisions();
 
+    manager.register_pass<ov::pass::MarkPrecisionSensitiveDivides>();
```

Comment:
Lets maybe have it in Common? GPU will need this code also we can add this into Common pipeline enabled by default after the first `ConstantFolding`. Plugins which don't support dynamism will have folded ShapeOf sub-graphs starting after such `ConstantFolding` but CPU (and GPU in the future) would have valid behavior (no Divide decomposition).

Reply:
Markup pass has been moved into common optimization pass. Thanks!